src/Stepper/Step.less

```diff
@@ -0,0 +1,174 @@
+@import "../less/fonts.less";
+@import (reference) "../less/index";
+
+@stepIconWidth: 1.75rem; // 28px
+@iconBorderWidth: @size_4xs; // 1px
+@iconFocusShadowThickness: @size_2xs; // 4px
+@titleFontSize: 1.25rem; // 20px
+@alertSize: 0.375rem; // 6px
+
+.Step {
+  .flexbox();
+  .alignItems(flex-start);
+  width: 100%;
+  .padding--y--2xs;
+}
+
+.Step--button {
+  .button--reset();
+  .padding--none;
+  text-align: left;
+  .text--line-height-3;
+}
+
+.Step--title {
+  overflow: hidden;
+  color: @neutral_gray;
+  font-size: @titleFontSize;
+  .text--semi-bold;
+  .margin--top--2xs;
+  -ms-flex: 0 1 auto;
+  transition: color @timingQuickly ease-out;
+}
+
+.Step--description {
+  color: @neutral_gray;
+  .text--medium;
```

Comment:
this is repeated a lot through this file. if it's common to most of the component, can move up to `.Step`. then it can be overridden wherever that's necessary (same for `.text--regular` below)